public/configs/otherMarketData.json

```diff
@@ -124,65 +102,52 @@
       "amount": "10000000000000000000000"
     },
     "meta": {
-      "assetName": "AAVE",
-      "referencePrice": 113.65607354685643
+      "assetName": "BOOK OF MEME",
+      "referencePrice": 0.009254383694412675
     }
   },
-  "ADA": {
-    "title": "Add ADA-USD perpetual market",
-    "summary": "Add the x/prices, x/perpetuals and x/clob parameters needed for a ADA-USD perpetual market. Create the market in INITIALIZING status and transition it to ACTIVE status after 3600 blocks. Added via the new market widget.",
+  "CELO": {
+    "title": "Add CELO-USD perpetual market",
+    "summary": "Add the x/prices, x/perpetuals and x/clob parameters needed for a CELO-USD perpetual market. Create the market in INITIALIZING status and transition it to ACTIVE status after 3600 blocks. Added via the new market widget.",
     "params": {
       "id": 2,
-      "ticker": "ADA-USD",
+      "ticker": "CELO-USD",
       "marketType": "PERPETUAL_MARKET_TYPE_CROSS",
       "priceExponent": -10.0,
-      "minPriceChange": 2500,
+      "minPriceChange": 4000,
       "minExchanges": 3,
       "exchangeConfigJson": [
         {
           "exchangeName": "Binance",
-          "ticker": "ADAUSDT",
-          "adjustByMarket": "USDT-USD"
-        },
-        {
-          "exchangeName": "Bitstamp",
-          "ticker": "ADA/USD"
-        },
-        {
-          "exchangeName": "Bybit",
-          "ticker": "ADAUSDT",
+          "ticker": "CELOUSDT",
           "adjustByMarket": "USDT-USD"
         },
         {
           "exchangeName": "CoinbasePro",
-          "ticker": "ADA-USD"
+          "ticker": "CGLD-USD"
         },
         {
-          "exchangeName": "Huobi",
-          "ticker": "adausdt",
+          "exchangeName": "Gate",
+          "ticker": "CELO_USDT",
           "adjustByMarket": "USDT-USD"
         },
-        {
-          "exchangeName": "Kraken",
-          "ticker": "ADAUSD"
-        },
         {
           "exchangeName": "Kucoin",
-          "ticker": "ADA-USDT",
+          "ticker": "CELO-USDT",
           "adjustByMarket": "USDT-USD"
         },
         {
           "exchangeName": "Mexc",
-          "ticker": "ADA_USDT",
+          "ticker": "CELO_USDT",
           "adjustByMarket": "USDT-USD"
         },
         {
           "exchangeName": "Okx",
-          "ticker": "ADA-USDT",
+          "ticker": "CELO-USDT",
```

Comment:
[q] did we cut over all our cardano pairs to celo? i don't have a lot of context so not sure if we're expecting a lot of tickers to change

Reply:
cardano is a live market so it got filtered out, CELO happens to be the next market alphabetically so it looks like it got replaced but it's just deletion of cardano and addition of celo!